src/Framework/Traits.cs

```diff
@@ -316,11 +316,6 @@ public bool LogTaskInputs
         /// </summary>
         public readonly bool DisableSdkResolutionCache = Environment.GetEnvironmentVariable("MSBUILDDISABLESDKCACHE") == "1";
 
-        /// <summary>
-        /// Disable the NuGet-based SDK resolver.
-        /// </summary>
-        public readonly bool DisableNuGetSdkResolver = Environment.GetEnvironmentVariable("MSBUILDDISABLENUGETSDKRESOLVER") == "1";
-
```

Comment:
Removing an unused field here.

Reply:
Should we just remove it, or should we make it do something? I imagine someone wanted it at some point, so I'm curious why making it not do anything is ok.

Reply:
This is likely a leftover from when the NuGet resolver was part of this repo. The environment variable is read in the resolver and presumably doing what it was doing here:
https://github.com/NuGet/NuGet.Client/blob/566e2853b8537e8b484844fad32b3f6073ba3167/src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/NuGetSdkResolver.cs#L34